### DISCUSSION TITLE
建议提升webdav兼容性，很多webdav工具都不兼容报错。

### DISCUSSION THREAD

**Discussion by zjw345** [AlistGo/alist#643] [Q&A]
建议提升webdav兼容性，很多webdav工具都不兼容报错
![未标题-1](https://user-images.githubusercontent.com/35001234/156096495-719826e6-bbea-4941-b97d-9629fa70029e.jpg)
。

**Comment by xhofe** [comment]
我没有nas。且无精力兼容所有客户端。

**Reply by zjw345** [reply]
webdav不是通用标准？
有没有一种方法，可以简单、直接兼容所有webdav工具？
那么好的工具，希望越做越好。

**Comment by gzxiexl** [comment]
https://github.com/messense/aliyundrive-webdav
这个库的WebDAV用起来非常稳定，是否可以更换一个WebDAV 服务端的代码？本人不懂编程，纯粹提个建议
